sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormTableCell.cs

```diff
@@ -2,14 +2,15 @@
 // Licensed under the MIT License.
 
 using System.Collections.Generic;
+using System.Linq;
 
 namespace Azure.AI.FormRecognizer.Models
 {
     /// <summary>
     /// </summary>
     public class FormTableCell : FormContent
     {
-        internal FormTableCell(DataTableCell_internal dataTableCell, ReadResult_internal readResult, IReadOnlyList<string> references)
```

Comment:
Removed the extra parameter from the ctor since we can get it from `dataTableCell`.